src/land/afwa_snod2ioda.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+#
+# (C) Copyright 2020 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+import pygrib
+import time, os, sys
+import argparse
+import netCDF4 as nc
+import numpy as np
+from datetime import datetime, timedelta
+from pathlib import Path
+
+IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
+if not IODA_CONV_PATH.is_dir():
+    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
+sys.path.append(str(IODA_CONV_PATH.resolve()))
+
+import ioda_conv_ncio as iconv
+from orddicts import DefaultOrderedDict
+
+vName = {
+    'A': "afwa_snow_depth",
+}
+
+locationKeyList = [
+    ("latitude", "float"),
+    ("longitude", "float"),
+    ("datetime", "string")
+]
+
+AttrData = {}
+
```

Comment:
Hi Greg, look at all python code in marine and chem, they do not need such a sentence. I googled and the message told me that all computer sever will set UTC. If you want to set a specific time, you can use os.environ["TZ"] = "America/New_York".  I am not sure if this is needed either.

Reply:
Not sure that either case described needs the time zone explicitly set. Most of these converters take the timestamps from the input files which are almost certainly UTC, if not, then those specific converters would need to convert to UTC. Only need to worry about this if the converter is creating the timestamp somehow (and you can force utctime in python, which probably is the preferred way rather than setting the system environment variable)

Reply:
Here is the *exact* type of case that setting the env to UTC time zone is required.
`data['ob_datetime'].append(datetime.fromtimestamp(utime).strftime("%Y-%m-%dT%H:%M:%SZ"))`

Since you're not explicitly using a datetime.Method.strftime, you don't see why I am suggesting it.  So it is OK in your case, but if one does any date manipulations inside the code, my suggested fix is highly important.  Took Steve H and I nearly half-hour to discover this behavior.  It could be considered a "safety net" in all our codes handling dates, but I will relent if you don't want to use it.  I plan to use it regardless since I am too worried I will forget and do a date manipulation some day and get bitten again.

Reply:
yes exactly, but one could/should probably use datetime.fromtimestamp(timestamp, timezone.utc) rather than having the script modify the machine's environment.

Reply:
Cory:  Unfortunately Steve and I tried *exactly* that idea but it failed; hence we found via google search the os setting - which only holds as long as the python script runs so I'm not following what's the harm here.  Regardless, I relent this point and I'll just keep using it myself as a safety net.

Reply:
Ok no problem, no harm at all, thanks for clarifying. You're right, I forgot that it only would affect it for child processes of this python script. I didn't realize that option I provided wouldn't work. It seems we are all constantly learning.

We are on the same page here, we need to make things consistent and standard, whether it be units, variable names, or time zones.

Reply:
Okay, I can put this line before the sentence "class AFWA(object):" as it does not harm but secured. Thank you, Greg.